src/imagePickerIOS.rei

```diff
@@ -0,0 +1,14 @@
+let canUseCamera: (unit => unit) => unit;
+
+let canRecordVideos: (unit => unit) => unit;
+
+type error = {
+  .
+  "code": int,
+  "message": string,
+};
+
+let openCameraDialog: (~videoMode: bool, unit => unit, error => unit) => unit;
```

Comment:
I think all these args should be labelled. WDYT?

Reply:
Yeah, that's good idea. Are onSuccess and onError good labels?

Reply:
yes! It'd be cool do it for both open* functions

Reply:
Yeah, it should be done right now.